cincinnati/src/plugins/internal/channel_filter.rs

```diff
@@ -41,6 +42,25 @@ impl ChannelFilterPlugin {
 
         Ok(Box::new(plugin))
     }
+
+    /// Takes a HashSet of release names and removes the corresponding conditional edges
+    /// containing the releases.  
+    pub fn remove_conditional_edges(
+        &self,
+        ce: &mut Vec<cincinnati::ConditionalEdge>,
+        releases: HashSet<String>,
```

Comment:
These are 2 different functions with different params, so cannot move to commons. 
Actually, we don't need this function but created it so that the existing function does not become too long and cluttered.